hub/powertoys/powerrename.md

```diff
@@ -110,6 +110,21 @@ A _Replace with_ text `Image_${padding=4;increment=2;start=10}_` would produce t
 - Image_0012_b.jpg
 - Image_0014_bc.jpg
 
+### Random string values
+
+If selected, you can use the following patterns as part of the _Replace with_ text:
+
+| Variable pattern    | Explanation                                                                                |
+| :------------------ | :----------------------------------------------------------------------------------------- |
+| `${rstringalnum=X}` | Random string with uppercase letters, lowercase letters and 0-9 digits, customized length. |
+| `${rstringalpha=X}` | Random string with uppercase letters and lowercase letters, customized length.             |
+| `${rstringdigit=X}` | Random string with 0-9 digits, customized length.                                          |
+| `${ruuidv4=X}`      | Random UUID according to v4 specification.                                                 |
```

Comment:
Why `=X`? Isn't it fixed length by definition?

Reply:
You are correct, good catch. Updated